src/main/java/com/google/sps/servlets/TripServlet.java

```diff
@@ -41,6 +52,22 @@
 @WebServlet("/calculate-trip")
 public class TripServlet extends HttpServlet {
 
+  // Create the GeoApiContext object.
+  private GeoApiContext context;
+  private int photoSrcSize = 400;
```

Comment:
photoSrcSize could be a constant since it's never updated

Reply:
Good point, thanks -- done!